server/models/investment.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
+
+const investmentSchema = new mongoose.Schema({
+  investorID: {
+    type: { type: Schema.Types.ObjectId, ref: "User" },
```

Comment:
Just wondering if type path should be wrapping in an array such as  ' type: [ { type: Schema.Types.ObjectId, ref: "User" } ] ' ?
The reason being, it should be able to list an array of investments made by the user

Reply:
each investment object would be an investment created by a user, so there wouldn't be a list of investor IDs because it's only created by only the one user.